src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableWriter.class.php

```diff
@@ -171,6 +236,8 @@ public function _RenameRelatedTables($sOldName, $sNewName)
      *
      * @param  $sOldTableName
      * @param  $sNewTableName
+     *
+     * @throws TPkgCmsException_Log
```

Comment:
Technically it's correct to add these declarations, but I'm very unsure if we should do this. The exception class is deprecated and usages should be removed. The database methods that throw this exception are deprecated as well, so they should be replaced by modern database calls. If we add the exception declarations, we entrench this unsatisfying intermediate state, so I'm inclined to say that it's better to let these methods throw undeclared exceptions instead.

Reply:
No changes yet.

Reply:
Removed (3 more)